### PR TITLE
Cleanup dot files

### PR DESCRIPTION
**Pull request by kawikabader**
## Summary

There are a lot of unnecessary dot files cluttering up packages.
## Changelog

Cleaned up unnecessary dot files.
## Test Plan

All tests are passing.

### CODE REVIEW

File: packages/gasket-log/package.json [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
+  "main": "src/server.js",
+  "browser": "lib/client.js",
+  "module": "src/client.js",
+  "files": [
+    "lib",
+    "src"
+  ],
   "scripts": {
-    "lint": "eslint *.js test/*.js",
+    "lint": "eslint src/*.js test/*.js",
```
Comment (kinetifex) [inline packages/gasket-log/package.json:13]:
```suggestion
    "lint": "eslint .",
```

And add below:
```json
  "eslintIgnore": [
    "lib",
    "coverage"
  ],
```

File: packages/gasket-redux/package.json [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
-  "module": "./src",
+  "main": "lib",
+  "browser": "lib",
+  "module": "src",
+  "files": [
+    "lib",
+    "src"
+  ],
   "scripts": {
     "lint": "eslint src __mocks__",
```
Comment (kinetifex) [inline packages/gasket-redux/package.json:13]:
```suggestion
    "lint": "eslint .",
```

And add below:
```json
  "eslintIgnore": [
    "lib",
    "coverage"
  ],
```

**Review by kinetifex — APPROVED** [review]
Great work aligning things. I found a couple of consistency tune-ups, then 👍🏻

**Review by rmarkins-godaddy — APPROVED** [review]

File: packages/gasket-plugin-redux/package.json [hunk 3]
```diff
@@ -4,6 +4,11 @@
   "description": "Gasket Redux Setup",
   "main": "lib",
   "types": "lib/index.d.ts",
+  "files": [
+    "generator",
+    "lib",
+    "!*.spec.js"
```
Comment (runruh-godaddy) [inline packages/gasket-plugin-redux/package.json:10]: we should probably create a backlog item to standardize to /test as the other packages are

File: packages/gasket-redux/package.json [hunk 4]
```diff
@@ -75,6 +79,10 @@
       "unicorn/filename-case": "error"
     }
   },
+  "eslintIgnore": [
+    "lib",
+    "coverage"
```
Comment (runruh-godaddy) [inline packages/gasket-redux/package.json:84]: nit: should we standardize on this?

**Review by runruh-godaddy — APPROVED** [review]

File: packages/gasket-redux/package.json (hunk 4 above)
Comment (kinetifex) [reply packages/gasket-redux/package.json:84]: These may be the last couple outliers, so yes I think standardizing on `"eslint ."` while using `eslintIgnore` will be complete.

**Pull request merged**